x-pack/packages/kbn-langchain/server/language_models/simple_chat_model.ts

```diff
@@ -95,7 +99,7 @@ export class ActionsClientSimpleChatModel extends SimpleChatModel {
       throw new Error('No messages provided.');
     }
     const formattedMessages = [];
-    if (messages.length === 2) {
+    if (messages.length >= 2) {
```

Comment:
This change looks fine to me, but I was curious why it was originally locked to `2` exactly before so I checked with @stephmilovic and it seems to be an original implementation artifact and she agreed it should be 👍 as well.